CogniteSdk/test/fsharp/TimeSeries.fs

```diff
@@ -484,4 +484,60 @@ let ``Synthetic Query with Error is Ok`` () = task {
     // Assert
     test <@ ts |> Seq.length > 0 @>
     test <@ ts |> Seq.forall (fun x -> x.Error |> (not << isNull)) @>
+}
+
+[<Fact>]
+let ``Interact with timeseries using the new unit capabilities is Ok`` () = task {
+    // Arrange
+    let externalIdString = Guid.NewGuid().ToString();
+    let unitExternalId = "temperature:deg_c"
+    let unitQuantity = "Temperature"
+    let dto =
+        TimeSeriesCreate(
+            ExternalId = externalIdString,
+            Name = "Create Timeseries sdk test",
+            Description = "dotnet sdk test",
+            IsStep = false,
+            UnitExternalId = unitExternalId
+        )
+    let query1 =
+        TimeSeriesSearch(
+            Filter = TimeSeriesFilter(UnitExternalId = unitExternalId),
+            Limit = Nullable 10
+        )
+    let query2 =
+        TimeSeriesSearch(
+            Filter = TimeSeriesFilter(UnitQuantity = unitQuantity),
+            Limit = Nullable 10
+        )
+    let updateDto =
+        TimeSeriesUpdateItem(
+            externalId = externalIdString,
+            Update = TimeSeriesUpdate(
+                UnitExternalId = UpdateNullable("temperature:deg_f")
+            )
+        )
+
+    // Act
+    let! timeSereiesResponses = writeClient.TimeSeries.CreateAsync [ dto ]
```

Comment:
are you planning on testing the datapoints query as well here?

Reply:
I'm working on it 😅

Reply:
And it's done